Controller/Adminhtml/Importcategory/Save.php

```diff
@@ -300,36 +360,53 @@ protected function _getKeyValue($row, $headerArray)
         return $temp;
     }
 
-    protected function _getImagePath($categoryimage)
+    /**
+     * Fetches image from URL and inserts it into the image path directory
+     *
+     * @param $categoryImage
+     * @return mixed
+     */
+    protected function getImagePath($categoryImage)
     {
-        $weburl = strpos($categoryimage, 'http://');
-        if ($weburl!==false) {
-            $imagepath = $this->_filesystem->getDirectoryRead(DirectoryList::MEDIA)
+        $webUrl = strpos($categoryImage, 'http://');
+
+        if ($webUrl !== false) {
+            $imagePath = $this->_filesystem->getDirectoryRead(DirectoryList::MEDIA)
                 ->getAbsolutePath('catalog/category');
-            $this->_fileio->mkdir($imagepath, '0777', true);
-            $file = file_get_contents($categoryimage);
-            if ($file!='') {
+            $this->_fileio->mkdir($imagePath, '0777', true);
+            $file = file_get_contents($categoryImage);
+
+            if ($file != '') {
                 $allowed =  ['gif','png' ,'jpg', 'jpeg'];
-                $ext = strtolower(pathinfo($categoryimage, PATHINFO_EXTENSION));
+                $ext = strtolower(pathinfo($categoryImage, PATHINFO_EXTENSION));
+
                 if (in_array($ext, $allowed)) {
-                    $imagename = pathinfo($categoryimage, PATHINFO_BASENAME);
-                    if (!is_dir($imagepath)) {
-                        $this->_fileio->mkdir($imagepath, '0777', true);
-                        $this->_fileio->chmod($imagepath, '0777', true);
+                    $imageName = pathinfo($categoryImage, PATHINFO_BASENAME);
+
+                    if (!is_dir($imagePath)) {
+                        $this->_fileio->mkdir($imagePath, '0777', true);
+                        $this->_fileio->chmod($imagePath, '0777', true);
                     }
-                    $imagepath = $imagepath.'/'.$imagename;
-                    $result = file_put_contents($imagepath, $file);
+                    $imagePath = $imagePath.'/'.$imageName;
+                    $result = file_put_contents($imagePath, $file);
+
                     if ($result) {
-                        return $imagename;
+                        return $imageName;
                     }
                 }
             }
         } else {
-            return $categoryimage;
+            return $categoryImage;
         }
     }
 
-    protected function getparentCategories($category)
+    /**
+     * @param $category
+     * @return \Magento\Framework\DataObject[]
+     * @throws \Magento\Framework\Exception\LocalizedException
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
```

Comment:
As with previous comment - use `use` statements.

Reply:
it this sis the only place this is used then no need to declare a `use`